quickstart/101-front-door-standard-premium/providers.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~> 3.27.0"
+      version = "~> 3.67.0"
```

Comment:
@johndowns Why are you restricting the provider version to 3.67.x instead of using "~>3.0", which uses 3.x   ?

Reply:
Originally I think we followed along with the other samples, which seem to use this approach (or did at the time, anyway).

I'd be happy to change this to `~> 3.0` if that's the standard - however, given there's been a breaking change somewhere between 3.27 and 3.67 (and hence the changes to app-service.tf in this PR), I assumed it's best to specify a version to avoid breaking changes affecting the sample.

Please let me know what you'd like me to do and I'll adjust as required.